state-recover/appcore/clients-interfaces/build.gradle

```diff
@@ -0,0 +1,11 @@
+plugins {
+  id 'net.consensys.zkevm.kotlin-library-conventions'
+}
+
+group = 'build.linea.staterecover'
```

Comment:
need to specify this because if this gradle [issue](https://discuss.gradle.org/t/multi-module-project-with-sub-modules-with-same-name/31928)